pillar.example

```diff
@@ -96,7 +96,7 @@ users:
     gitconfig:
       user.name: B User
       user.email: buser@example.com
-      url."https://".insteadOf: "git://"
+      "url.https://.insteadOf": "git://"
```

Comment:
It's true that the underlying syntax for gitconfig would be correct, but it doesn't end up like that in the file.

The old example would yield
```
[url "\"https://\""]
        insteadOf = git://
```
and if you encase the whole string (new code) it's
```
[url "https://"]
        insteadOf = git://
```
The quotes in the example are for salt to not stumble over special characters (like ':'), not for gitconfig.